src/utils/tokens.ts

```diff
@@ -7,3 +9,11 @@ export const getColonyTokenAddress = async (
   const tokenAddress = await colonyClient.getToken();
   return tokenAddress;
 };
+
+export const getColonyTokenData = async (
+  tokenAddress: string,
+  colonyClient: AnyColonyClient,
+): Promise<{ name: string; decimals: number; symbol: string }> => {
+  const tokenClient = await getTokenClient(tokenAddress, colonyClient.provider);
+  return await tokenClient.getTokenInfo();
+};
```

Comment:
Is this used anywhere? I can see it imported in `utils/motions` but doesn't seem to be called